src/script/standard.h

```diff
@@ -157,9 +156,9 @@ std::string GetTxnOutputType(txnouttype t);
  *
  * @param[in]   scriptPubKey   Script to parse
  * @param[out]  vSolutionsRet  Vector of parsed pubkeys and hashes
- * @return                     The script type. TX_NONSTANDARD represents a failed solve.
+ * @return                     The script type. NONSTANDARD represents a failed solve.
```

Comment:
nit:
```suggestion
 * @return                     The script type. TxoutType::NONSTANDARD represents a failed solve.
```

Reply:
I think in comments we don't need to be overly verbose. Especially not in the file that defines these values and where the return type is mentioned one line below the comment. I think the comment is clearer as is.